src/context/SyntheticsEvents/SyntheticsEventsProvider.tsx

```diff
@@ -138,6 +154,24 @@ export function SyntheticsEventsProvider({ children }: { children: ReactNode })
     OrderExecuted: (eventData: EventLogData, txnParams: EventTxnParams) => {
       const key = eventData.bytes32Items.items.key;
 
+      const order = orderStatuses[key]?.data;
+
+      if (order) {
+        const metricId = isSwapOrderType(order.orderType)
+          ? getSwapOrderMetricId(order)
+          : getPositionOrderMetricId(order);
+
+        const metricData = metrics.getPendingEvent(metricId, true);
+        const metricType = metricData?.metricType || "unknownOrder";
+
+        metrics.sendMetric({
+          event: `${metricType}.executed`,
```

Comment:
sorry, I now see difference between metricType and event

let's make it separate?
```
event: 'executed' | 'failed' | etc
metricType: MetricType
```